addons/account_generator/models/cleaner.py

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+
+from odoo import api, models, fields
+
+
+class AccountCleaner(models.TransientModel):
+    _name = "account.cleaner"
+    _description = "Account Cleaner"
+
+    company_ids = fields.Many2many('res.company', default=lambda self: self.env.companies)
+
+    def clean_account_move(self):
+        company_ids = tuple(self.company_ids.ids)
+
+        self.env.cr.execute('delete from account_partial_reconcile where company_id in %s;', [company_ids])
+        self.env.cr.execute('delete from account_move_line where company_id in %s;', [company_ids])
+        self.env.cr.execute('delete from account_move where company_id in %s;', [company_ids])
```

Comment:
Can't this cleaner stuff be a little too dangerous when ran on a production db? The simple fact people could install that themselves gets me worried... Or maybe with something prompting the user like "are you really really 100% super certain that you want to do that?". But even so, I find it dangerous.